theme/less/gouvfr-dataset.less

```diff
@@ -210,68 +210,6 @@ body.dataset-display {
         }
     }
 
-    .reuses-list.in-modal {
```

Comment:
cf https://github.com/etalab/udata-gouvfr/pull/228#pullrequestreview-73756697, I don't think those styles should be removed.